### PR TITLE
feat(KTL-1544): add Optimizely script to index

### PR DESCRIPTION
**Pull request by zoobestik**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -241,6 +241,8 @@ function Index() {
             <ThemeProvider theme={'dark'}>
                 <GlobalFooter />
             </ThemeProvider>
+
+            <Script src={'https://cdn.optimizely.com/js/26633200186.js'} />
```
Comment (nikpachoo) [inline pages/index.tsx:245]: I expected to see the script at the beginning of the layout.
What do you think?

Comment (zoobestik) [reply pages/index.tsx:245]: In next try 😀

Comment (nikpachoo) [reply pages/index.tsx:245]: What do you mean? I see the branch was force-pushed, but no changes has been made.

Comment (zoobestik) [reply pages/index.tsx]: Fixed i added `strategy={"beforeInteractive"}`

**Pull request merged**